queries/xresources/folds.scm

```diff
@@ -0,0 +1,10 @@
+[
+  (comment)+
+  (elif_directive)
+  (elifdef_directive)
+  (else_directive)
+  (if_directive)
+  (ifdef_directive)
+  (include_directive)+
+  (preprocessor_comment)
```

Comment:
Would you mind moving the quantified captures to the end of this alternation? You shouldn't have to, but I've noticed a weird but in tree-sitter where folds of different node kinds in the alternations are merged if they are a quantified one followed by a regular one...